### PR TITLE
[Ember-Migration] [Tests] Scroll to Top

### LINKED ISSUE #376: [Ember-Migration] [Tests] Scroll to Top
## What to do?
- Write tests for scroll-to-top component

## Note 
- PR should be raised to `develop-ember` not `develop`
- Create a separate branch `test/scroll-to-top` from `develop-ember` and write the tests in `scroll-to-top-test.js`

### PR DESCRIPTION
**Pull request by ranitasahaa1230**
### What is the change?

- Added tests for the scroll-to-top component

### Is Development Tested?

- [x] Yes

### CODE REVIEW

File: tests/integration/components/scroll-to-top-test.js [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
 module('Integration | Component | scroll-to-top', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders', async function (assert) {
-    assert.expect(0);
+    assert.expect(1);
+
+    await render(hbs`<ScrollToTop />`);
+    
+    await click('[data-test-scroll-to-top]');
```
Comment (rohan09-raj) [inline tests/integration/components/scroll-to-top-test.js:14]: Can we please test for the action that is being performed at the click of this button?
This means when the button is clicked, the page scrolls to position 0 (check the `scroll-to-top.js` for reference)
that is needed to be tested here.

File: tests/integration/components/scroll-to-top-test.js [hunk 2]
```diff
@@ -1,10 +1,17 @@
...
 
 module('Integration | Component | scroll-to-top', function (hooks) {
   setupRenderingTest(hooks);
 
   test('it renders', async function (assert) {
-    assert.expect(0);
+    assert.expect(1);
+
+    await render(hbs`<ScrollToTop />`);
+
```
Comment (rohan09-raj) [inline tests/integration/components/scroll-to-top-test.js:13]: Please check why the GitHub actions are failing

**Review by rohan09-raj — CHANGES_REQUESTED** [review]

File: tests/integration/components/scroll-to-top-test.js (hunk 1 above)
Comment (rohan09-raj) [reply tests/integration/components/scroll-to-top-test.js:14]: You can test the components in the following steps :
- Scroll the window a little bit using `scrollTo`
- Click the `scroll-to-top` button 
- Write an assert  to check the height that it is `0` or not.

File: tests/integration/components/scroll-to-top-test.js [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
   setupRenderingTest(hooks);
 
-  test('it renders', async function (assert) {
-    assert.expect(0);
+  test('scroll to top renders', async function (assert) {
+    assert.expect(1);
+
+    await render(hbs`<ScrollToTop />`);
+    
+    await scrollTo('#scroll__btn', 0, 100);
```
Comment (satyam73) [inline tests/integration/components/scroll-to-top-test.js:14]: Use data-test selector here too.

**Comment by rohan09-raj** [comment]
PR became stale and the author is not working on it anymore. Closing the PR and reassigning the issue

**Pull request closed without merging**